accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -687,34 +687,56 @@ It's a compile-time error if:
     class M with S {} // Error, for several reasons.
     ```
 
-*   A class extends or mixes in a declaration marked `interface` or `final`
+*   A declaration has a direct super declaration from another library
+    which is marked `final`.
```

Comment:
Should we add the `_(with some exceptions for platform libraries)_` here too?

Reply:
Of course, done.